examples/classic/distributed/TwoPhaseCommit/two_phase_commit.qnt

```diff
@@ -0,0 +1,162 @@
+// -*- mode: Bluespec; -*-
+
+/* This specification stems from the specification of the two phase protocol
+   by Leslie Lamport:
+
+   https://github.com/tlaplus/Examples/blob/master/specifications/transaction_commit/TwoPhase.tla */
+
+module two_phase_commit {
+  type ResourceManagerState = Working | Prepared | Committed | Aborted
+  var resourceManagerStates: str -> ResourceManagerState
+
+  type TransactionManagerState = Init | Done
+  var transactionManagerState: TransactionManagerState
```

Comment:
Two questions here, comparing with https://github.com/tlaplus/Examples/blob/68aec27a9bdfd4085ee02c3154e39e8094ae347a/specifications/transaction_commit/TwoPhase.tla#L58

1. Why rename the "Commited" to "Done"? I'd suggest keeping "Committed" and if adding a change, rephrase "Init" to "Initiated".
2. Why is there no `Aborted` state?